core/src/main/java/io/github/wysohn/triggerreactor/core/IEventHook.java

```diff
@@ -1,6 +1,7 @@
 package io.github.wysohn.triggerreactor.core;
 
-@FunctionalInterface
 public interface IEventHook {
+    int getPriority();
```

Comment:
```suggestion
    default int getPriority() {
        return -1;
    }
```
Add `default` access modifier so we can keep using `FunctionalInterface`